File: app/src/main/java/ru/arhlib/app/services/ContactsActivity.java
package ru.arhlib.app.services;

import android.os.Bundle;
import android.view.MenuItem;
import android.webkit.WebView;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import ru.arhlib.app.MyWebViewClient;
import ru.arhlib.app.R;

public class ContactsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.webview);

        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
        }

        WebView webView = findViewById(R.id.webview);
        webView.setWebViewClient(new MyWebViewClient());
        webView.loadUrl("file:///android_asset/contacts.html");
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                onBackPressed();
                return true;
            default:
                return super.onOptionsItemSelected(item);
        }
    }
}

File: app/src/main/java/ru/arhlib/app/ContactsActivity.kt
package ru.arhlib.app

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import ru.arhlib.app.browser.WebViewClientWithCustomTabs
import ru.arhlib.app.databinding.WebviewBinding

class ContactsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = WebviewBinding.inflate(layoutInflater)
        setContentView(binding.webview)

        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        binding.webview.webViewClient = WebViewClientWithCustomTabs()
        binding.webview.loadUrl("file:///android_asset/contacts.html")
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        return when (item.itemId) {
            android.R.id.home -> {
                onBackPressed()
                true
            }
            else -> super.onOptionsItemSelected(item)
        }
    }
}
